functions.py

```diff
@@ -0,0 +1,70 @@
+from math import remainder
+import random
+from datetime import datetime
+from typing import List
+
+from .readJSON import Myjson
+from .HappyApp import HappyApp
+from config import day, _Team, MessageType
+from mail import Mail
+
+def random_lst_gen(n : int):
+    """
+    input: n - a positive integer number
+    output: a random sequence of numbers from 1 to n
+            sequence has all unique numbers
+    """
+    lst = list(range(n))
+    rnd_lst = []
+    for _ in range(n):
+        i = random.randint(0,len(lst) - 1)
+        rnd_lst.append(lst.pop(i))
+    return rnd_lst
+
+def isToday(d : day, time = None):
+    """
+    Input: @time - integer constant indicating an hour 0 - 23
+           @d - day enum 
+    Output: True if current time matches day and hour, else False.
+    """
+    if time == None:
+        return datetime.today().weekday() == day.value
+    else:
+        return datetime.today().weekday() == day.value and \
+            time == datetime.today().hour
+
+def scramble(lst : List):
+    """
+    Input: lst of strings
+    Output: scrambled list according to indexes given by 'random_lst_gen'
+    """
+    rnd_lst = random_lst_gen(len(lst))
```

Comment:
use random.shuffle() for code readability and declarative usage instead of imperative lambda expressions.